fluent-package/test-verify-repo.sh

```diff
@@ -52,7 +60,12 @@ if [ -z "$REPO_TARGETS" ]; then
     REPO_TARGETS="exp/5 exp/lts/5"
 fi
 
-RESULTS=""
+echo "DEB_TARGETS: $DEB_TARGETS"
+echo "RPM_TARGETS: $RPM_TARGETS"
+echo "REPO_TARGETS: $REPO_TARGETS"
+sleep 3
+echo RESULTS=""
```

Comment:
> sleep 3
> echo RESULTS=""

Could you tell me the purpose of these two lines?

Reply:
unexpectedly garbage was injected.